inst/components/scss/sidebar.scss

```diff
@@ -35,7 +44,7 @@
     grid-column: 2 / 3;
     border-top-left-radius: 0;
     border-bottom-left-radius: 0;
-    padding: $bslib-sidebar-padding;
+    padding: var(--bslib-sidebar-padding);
```

Comment:
I'm glad you worked these kinds of changes into this PR! I'd be helpful to call this out in the PR description and explain our new approach.

Reply:
Good point, done